### DISCUSSION TITLE
Script fails to compile

### DISCUSSION THREAD

**Discussion by edwardr** [vercel/next.js#13416] [Help] [answered]
# Bug report

## Describe the bug

Assignment of object in script fails to compile.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

Using  next/head in any page:

```
<script src="https://cdn.broadstreetads.com/init-2.min.js" async></script>
<script>
	window.broadstreet = window.broadstreet || { run: [] }
	window.broadstreet.run.push(function() {
	broadstreet.watch()
})
</script>
```

## Expected behavior

Successful compile 

## Screenshots

https://i.imgur.com/lyylTlp.png

## System information

- OS: Ubuntu
- Browser (if applies) Chromium
- Version of Next.js: 9.4
- Version of Node.js: 12.12.0

## Additional context

Unexpected token, expected "}" at "run:"
`{ run: [] }`

**Comment by Timer** [comment, marked as answer]
This is a general React question and requires [`dangerouslySetInnerHTML`](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml).

**Reply by Timer** [reply]
Try:

```jsx
    <script src="https://cdn.broadstreetads.com/init-2.min.js" async></script>
    <script
      dangerouslySetInnerHTML={{
        __html: `
		  window.broadstreet = window.broadstreet || { run: [] };
		  window.broadstreet.run.push(function () { broadstreet.watch(); });
        `,
      }}
    />
```
